phpcs.ruleset.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ruleset name="WP-API">
+	<description>Sniffs for the coding standards of the WP-API plugin</description>
+
+	<rule ref="WordPress-VIP">
```

Comment:
Looks like the VIP rules are too strict here; we're getting `get_term_by is prohibited, please use wpcom_vip_get_term_by() instead.`

I'm not familiar enough with the VIP sniffs, but can we keep the security/etc ones but ditch the VIP-only ones?